chart/templates/connection-secret-job.yaml

```diff
@@ -3,6 +3,7 @@
 ---
 apiVersion: v1
 kind: ServiceAccount
+automountServiceAccountToken: false
```

Comment:
This SA is used by the `{{ .Release.Name }}-connection-secret` which in turn starts a `password-initializer.sh` script which uses `kubectl` to contact the k8s API server. If we prevent mounting a token, the script will fail to work.
As such I think we should set this to `true` and put a comment why this is needed.

Reply:
👍